metadata-ingestion/src/datahub/ingestion/source/delta_lake/source.py

```diff
@@ -122,11 +122,6 @@ def __init__(self, config: DeltaLakeSourceConfig, ctx: PipelineContext):
             config_report,
         )
 
-    @classmethod
-    def create(cls, config_dict: dict, ctx: PipelineContext) -> "Source":
```

Comment:
the `config_class` decorator auto-generates this method, so we can just remove it here